extensions/common/vault/vault-azure/src/main/java/org/eclipse/edc/vault/azure/AzureVaultExtension.java

```diff
@@ -24,14 +24,23 @@
 import org.eclipse.edc.spi.security.Vault;
 import org.eclipse.edc.spi.system.ServiceExtension;
 import org.eclipse.edc.spi.system.ServiceExtensionContext;
+import org.eclipse.edc.spi.system.configuration.Config;
+import org.jetbrains.annotations.NotNull;
 
 @Extension(value = AzureVaultExtension.NAME)
 public class AzureVaultExtension implements ServiceExtension {
 
     public static final String NAME = "Azure Vault";
+
     @Setting("Name of the Azure Vault")
     private static final String VAULT_NAME = "edc.vault.name";
 
+    @Setting("If non blank, ignore edc.vault.name and use this value as an Azure vault.")
+    private static final String VAULT_OVERRIDE = "edc.vault.override";
```

Comment:
minor nit: since `edc.vault.name` refers only to the name, one could be mislead into thinking `edc.vault.override` also refers to the name, where in reality it must contain the url. 
I would suggest to rename the property to `edc.vault.url.override`, and perhaps also put some validation in place to assert the value is indeed a URL.

Reply:
Agree. Prepended both `override` and `override.unsafe` with `url.`